### PR TITLE
3678 authority dashboard

### PR DESCRIPTION
**Pull request by alibombaye**
added authority dashboard page

### CODE REVIEW

File: app/pages/authority/routes.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  sectionGroups: [
+    {
+      id: 'sectiongroup-id',
+      title: 'Sectiongroup Title',
+    },
+  ],
+};
+
+describe('GET /authority/solution/:solutionId', () => {
+  it('should return the correct status and text if there is no error', () => {
```
Comment (kat1906) [inline app/pages/authority/routes.test.js]: what about if there is an error?

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "lint:js": "eslint \"**/*.js\" --ignore-pattern node_modules/",
     "lint:css": "sass-lint 'app/**/*.scss' -v -q",
     "nuke": "npm run clean && rm -rf node_modules/",
-    "start:dev": "babel-node server",
+    "start:dev": "cross-env LOGGER_LEVEL=info babel-node server",
```
Comment (kat1906) [inline package.json]: did you want to leave this is here?

Comment (alibombaye) [reply package.json]: Not intentionally. But now that I see it. I quite like it so yeah let's leave it in there

Comment (kat1906) [reply package.json]: 👍

File: app/pages/authority/routes.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/authority/routes.test.js]: Good point. I've added a unit test to the main routes.test.js to check the error page is shown if there is an error from the /authority endpoint.

**Pull request merged**